test/e2e/eth1/deploy.test.ts

```diff
@@ -1,20 +1,30 @@
-import { assert } from "chai";
+import {assert} from "chai";
 import {ethers} from "ethers";
 import sinon from "sinon";
 
 import {Eth1Wallet, EthersEth1Notifier} from "../../../src/eth1";
 import {depositContract} from "../../../src/eth1/dev/defaults";
 import {PrivateEth1Network} from "../../../src/eth1/dev";
 import logger from "../../../src/logger/winston";
+import {LevelDB} from "../../../src/db";
+import level from "level";
 
 describe("Eth1Notifier - using deployed contract", () => {
 
   let eth1Notifier;
   let eth1Network;
   let depositContractAddress;
   let provider;
+  const dbLocation = "./.__testdb";
+  const testDb = level(
+    dbLocation, {
+      keyEncoding: 'binary',
+      valueEncoding: 'binary',
+    });
+  const db = new LevelDB({db: testDb});
```

Comment:
As an aside, it might be nice to have a MemDB for testing purposes

Reply:
We have, pouch implementation is in memory, although becuse of the way we are searching values it has nearly same performance but yeah, I will change it to pouch

Reply:
ok, no worries, not super important, just a thought